.dotnet/src/Generated/Models/CreateChatCompletionRequest.cs

```diff
@@ -1,9 +1,7 @@
 // <auto-generated/>
 
-#nullable disable
-
+using OpenAI.ClientShared.Internal;
```

Comment:
I put those "emitted shared source" files in this namespace -- models will need to reference them for reading/writing model content (i.e. serialization routines ;))